psi4/src/psi4/libfock/PKmanagers.cc

```diff
@@ -370,6 +370,7 @@ void PKManager::form_D_vec(std::vector<SharedMatrix> D, std::vector<SharedMatrix
 
     D_ = D;
     all_sym_ = true;
+    symmetric_.clear();
```

Comment:
@jgonthier I think this was the source of the symmetric issues. The vector wasnt being cleared whenever PK was being called so it reach many more elements than density matrices. This fixes up local test cases and the issue I was having, but please confirm.